### PR TITLE
accept exchange feed options in config

### PR DESCRIPTION
**Pull request by jinusean**
### Description of code - what bug does this fix / what feature does this add?

- [x] - Tested
- [x] - Changelog updated

Relevant update to cryptofeed's changes in https://github.com/bmoscon/cryptofeed/pull/574

`configy.yaml` now accepts exchange feed params.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -69,10 +76,9 @@ exchanges:
         ticker: [BTC-USD]
     COINGECKO:
         retries: -1
-        market_info: [BTC-USD, ETH-BTC, LINK-ETH]   
 
 # Where to store the data. Currently arctic, influx, elastic, and parquet are supported. More than one can be enabled
-storage: [arctic, influx]
+storage: [artic, influx]
```
Comment (bmoscon) [inline config.yaml:81]: arctic is spelled incorrectly here

File: config.yaml [hunk 2]
```diff
@@ -69,10 +76,9 @@ exchanges:
         ticker: [BTC-USD]
     COINGECKO:
```
Comment (bmoscon) [inline config.yaml:77]: just remove the whole coingecko block

**Review by bmoscon — CHANGES_REQUESTED** [review]
see other comments

File: config.yaml (hunk 2 above)
Comment (jinusean) [reply config.yaml]: Removed

File: config.yaml (hunk 1 above)
Comment (jinusean) [reply config.yaml]: Spelling fixed.

**Comment by jinusean** [comment]
Requested changes have been applied.

**Review by bmoscon — APPROVED** [review]

**Pull request merged**